zephyr/include/rtos/wait.h

```diff
@@ -11,34 +11,22 @@
 #include <stdint.h>
 #include <stdbool.h>
 #include <zephyr/kernel.h>
+#include <zephyr/sys/time_units.h>
 #include <rtos/timer.h>
 
-/* TODO: use equivalent Zephyr */
-static inline void idelay(int n)
+static inline void wait_delay_us(uint64_t us)
```

Comment:
nitpick - having this function at the bottom of this patch, where it was before, would both make this patch smaller (probably) and easier to read and keep the grouping of all `wait_delay*()` functions as now. Not a show-stopper of course, only if you need to update